### DISCUSSION TITLE
Problem of tuned raids from a non full-on-hardcore-guild

### DISCUSSION THREAD

**Discussion by Krateozshael** [chromiecraft/chromiecraft#6498] [Suggestions]
Hey!
Im Shael, raidlead of EU's raiding team for the past 2 Years.
I make this post to give my few thoughts about the current state of raiding from a guild that masters all content (SSC full clear in week 2), but isnt still fully happy with the decisions CC made regarding the raiding experience.

Problem we have is that CC shouldnt be a place of purely hardcore raiding. Sure we know that people want some difficulty.
Nonetheless we think CC should do a social experiment with one upcomming raidtier:
Release a raid with the tuned version AND untuned with the original values, lets see what people play. If people want a higher difficulty, they can choose to do so then, no?

Regarding that, i dont want to hear the argument of 'but then i want more loot or sth else'. This is no valid argument.
You first said you want harder content, because you want a challenge, but this should be the "bonus" you get from the challenging experience.
What i want to say: people only want harder content to get more loot for doing so. And this is BS and shouldnt find a place here, personally.

Still i understand the frustration comming from not to getting rewarded after going through more consums/time then the "normal raiders". So just reimplement the tokensystem from Gruul/Mag or just put one more tierpiece in the loot or something like this.

We thought its just healthier for guilds to be able to do the raiding content fully to be able to playtest and contribute to CCs cause.
I just suggest a place where every player of skill can enjoy the community feeling of progressing a raid in their pace.
Just give the option to play with the OG values and to a tuned version for the guilds that want a bit more challenging experience. Just reuse your tokensystem!

In love and high respect to the CC staff for their work,
Shael

**Comment by balleny** [comment]
How are people supposed to playtest mechanics and content, when bosses die within 20 seconds?
in an not tuned version, people will run kara with 5 players and 25 player raids with 10-15 players.
the average joe that you want to playtest content is not doing bug reports for raid content, they just consume.

there was a bigger survey not too long ago, and iirc correctly, the majority voted against a mythic mode, but that might have been more or less linked with the intend to add additional abilities similar to retail.

from my pov, a "doable" version tuned as it was in original content (relativ to a raid strength at original release) is a good default. You are not supposed to have everything on farm at day zero.
an additional mythic mode, with more challenging tuning, with additional non-tier loot drops would be my preferred option. it reduces, but not eliminates the randomness factor in getting loot that is a core characteristic of wow. the "shop" option is something i very much dislike. especially, if it allows you to buy items from a boss you may never beat in the "mythic" version (imo, every boss should have their own coin/token if there is a shop like option)

and to give another proposal, which I would heavily favor but not many will agree with:

release bosses tuned, not dropping any loot for 2 weeks. Use the first two weeks for fixing remaining issues, then add loot to the tuned version (and potentially adjust the tuning if necessary up or down).
leave the content like that but reduce the tuning every 2 weeks by 5% or 10%. That way, the tuning would be removed practically in time with the next content release.

**Comment by Krateozshael** [comment]
After some hours, days to rethink some points, here my additional thoughts:

To be fair i maybe exaggerated in some points

My main concerns come from the POV of our NA raid, where i am no part of, but hear the stories and Frustration of our NA-RL Elitecow.
They have a hard time to keep an consistent raidroster, have to bring pugs every week, and failing/having problems with even the easier bosses. There should be some sort of way that such groups/guilds can do the content aswell (which is by maybe not fully, but partially removing the tuning)

Also i may overstepped with wanting OG values, in my opinion Gruul and Mag had the perfect balance/tuning. The fights werent too long, somewhat engaging and not a complete steamroll. SSC now feels just like the Mythic Version of Gruul, which werent much fun to do, imo. 
Like wtf is fun about a 14 mill hp Morogrimm?! This is no engaging tuning, nor a challenge.

A big part of my concerns come from the organization of RR raids. We only have/had and will have ONE raid day, since our guild is a dad guild and cant simply do/want more raiddays. Looking at Hyjal and BT, i dont see that both raids are doable in our 3 our time frame, and we cant extend this raidtime, since our raidteam reaches from NA to russia (when we finish our raidingtime its already 1 am in russia, and for the central european countries we cant start earlier due to work of bunch of raiders)

As a comparison SSC and TK were cleared in about 1h 30 min, and when CC stated how they want to approach the tuning, they want the values to be equal to those in TBC classic. Ofc it doesnt have to be 1h 30 but approaching the 3 hour mark for all raidingteams (even those of the lesser skill) should be the goal)
And a 14 mill hp boss doesnt help with that

Regarding the tokensystem:
I think for a pserver its a really good addition to the game, since its mitigate the unwanted/unnessacary grind you would have to do. Lets give you an example: we've cleared magtheridon since week one and did him every week. Guess how many plate belts we got; 
Right 0. 
Having a challenging mode which gives the option to enjoy a harder content people want, get a little, not too overpowered, reward and just Kills the Anti-fun of RNJesus. As said, since we only raid one day and many of us dont want to do more outside this timeframe (mean it for irl reason or simply you want to Play sth else) you shouldnt go through the hazzle and raiding a content that is no longer fun to you, since you raided that stuff already for the past 4-6 months.
Therefore im highly-pro a tokensystem

I personally like the suggestion someone Made underneath my post saying "each week reduce the tuning by 5%" this way you give the weaker raids the ability to finish the raid at some point and after some time when the next raid is released, the tuning is gone and you have a fresh raid you can progress on, and can get the missing items from the older raid

**Reply by balleny** [reply]
tbf this reads more like an internal discussion for your guild on how to approach raids and not to a generic CC topic.

**Reply by ishidinho** [reply]
Because of the low population of the server, the current end game is not really accessible if you are not part of a stable raid guild. So, every discussion about accessibility should be considered.

You asked how mechanics should be play-tested if bosses die in 20secs. I wonder how bosses should be play-tested if not everyone can play them? I would really like to play the endgame with more characters, but I cannot since no PUGs ate forming.

Let's take Morogrim, a boss with one phase the whole time. If you can play-test the boss with 14mil HP in 12min, you can test him in the same way with half the HP in half of the time. Same for all the trash, it is just time consuming with so real sense.

I think the weekly time investment of a raid group like Rum Rage is a great benchmark for raid tuning. If they can clear the current tier content in under 3 hours (this blizzlike btw) everyone can do it if they are willing to invest at least one evening a week.

It is not about Rum Rage. It is about end game accessibility and keeping CC attractive.

Long story short, I support the system we had with Gruul and Magtheridon.
You just want to slap some bosses? Play normal. 
You want to improve week by week but safely clear the content? Increase the number of hardmodes every week, but safely clear the content by switching to normal when your raid time goes to an end.
You want full hardmode? Do them and get rewarded with some juicy tokens to get your well deserved best in slots faster.
Great side effect: This will increase the overall play-test time for AzerothCore. Simple math.

**Reply by balleny** [reply]
unfortunately, your reply kinda confirms the reasoning behind my text, but lets start from top to bottom:

> Because of the low population of the server, the current end game is not really accessible if you are not part of a stable raid guild.

Minor topic, but in the non-christmas week, there were multiple 5/6 clear pugs, some 3/6 pugs, etc. Given these were week #1 pugs, I think it is quite accessible. If there are multiple, half-full guild groups that prefer to not raid instead of filling up with randoms or pugs, then it is a self-made decision of that guild. Further, at some points, 25% of the online chars were in SSC, which I would think of quite astonishing. 

> Let's take Morogrim, a boss with one phase the whole time.

And this is why i mention that just because more people "could" report things, it is unlikely to happen. First of all, you would need to have some incentive to report stuff, **then you would need to know the bosses or read guides.** Otherwise, how to report issues if you arent aware of the issues in the first place? Morogrim has 2 phases, one before and one after 25% HP. (ref: https://www.icy-veins.com/tbc-classic/morogrim-tidewalker-guide-strategy-abilities-loot). People that want to put in effort, are doing so already and are creating bug reports or spend some time on the PTR. For the others, there is either not enough incentive (intrinsic or extrinsic), the wrong incentive or other reasons. Obviously, there are things that can hardly be tested, and test vs. production may also result in differences on CC's environment.

The t5 content will also stay available during t6 content, no one is obliged to clear the content on day 1 or week 1. I personally enjoy progressing through a dungeon and dont mind working 2-5 raid nights (also stretched across weeks) to learn and master an tightly tuned encounter - at least i enjoy that a lot more than playing a loot pinata mode.

1 mode or 2 modes, additional rewards, tokens or not. obviously there are a multitude of combinations possible - there wont be the one solution fitting all. Further, the decision making on CC by the team to several topics has been reasonable from some pov and unreasonable from other pov

**Reply by ishidinho** [reply]
@balleny Sadly, you stopped quoting me at my last paragraph. Can you please explain why every group of players I mentioned won’t be happy with my suggested solution?

**Reply by balleny** [reply]
there you go with my further cents:

> I think the weekly time investment of a raid group like Rum Rage is a great benchmark for raid tuning. If they can clear the current tier content in under 3 hours (this blizzlike btw) everyone can do it if they are willing to invest at least one evening a week.

Are you sure that the majority of guilds raiding on retail did clear the current content within 3h or less, 2 weeks after release? From what i can recall, many guilds struggled weeks on Vashj (and Kael) for that matter and didnt kill them til the first wave of nerfes hit. A tuning different from that seems less blizzlike to me. The content will be run also later and CC anyway stopped fixing content 2 weeks after release, but started to collect bug tickets only (examples include Majordomo, Ouro, Magtheridon, Netherspite)

> It is not about Rum Rage. It is about end game accessibility and keeping CC attractive.

I started to doubt that CC is about being or staying attractive. They are either not thinking ahead of topics (esp. PvP), put out content in blizzlike order (which means catchup dungeons will be less run/tested, compared to a shift in progression (e.g. ZA between T5 and T6, or ZG between MC and BWL)), etc.. But this is yet another discussion.


> Long story short, I support the system we had with Gruul and Magtheridon.

Yet, after the survey the result apparently was that the majority does not favor that system.

> You just want to slap some bosses? Play normal.

and to be honest, I would expect 0 bug reports from that group of players. at max, it would be some whining in discord or world chat about something they think is a bug but is actually blizzlike, they just havent experienced yet on any other private server.  (_not the nicest point of view but this is the experience i have had on chromie since we left the low level brackets and more people joined not because of chromies concept but because of the serversize_)


> You want to improve week by week but safely clear the content? Increase the number of hardmodes every week, but safely clear the content by switching to normal when your raid time goes to an end.
You want full hardmode? Do them and get rewarded with some juicy tokens to get your well deserved best in slots faster.

I know that what you are describing is practically the ICC way of content, kill in hardmode what you can, farm the rest in normal mode. This is a concept that doesnt really fit with TBC or TBCC.
I think its totally fair to make the content accessible, and given that you can go into SSC and progress on each boss with your guild, or join a pug and even do vashj, seems a good tuning to me.

> Great side effect: This will increase the overall play-test time for AzerothCore. Simple math.

I dont wanna spoil the premise here again but I very much would expect 0 bug reports from that target group.

and to pick something from the initial post again
> Nonetheless we think CC should do a social experiment with one upcomming raidtier:
Release a raid with the tuned version AND untuned with the original values, lets see what people play.

wasnt there are vote/concensus exactly for that? Stating what the "community" wants and what not? ( I didnt take the survey for reasons but i understood that as goal and result of the survey)

**Comment by EliteCow09** [comment]
As the NA raid lead for Rum Rage (and long time CC enjoyer), I'll add some context for why I support implementing normal and mythic raid modes again.

- **Balance.** I was originally against mythic dungeons and raids because of the customized loot. I was worried it would create a less authentic experience. Chasing random affixes for certain specs that would boost player power beyond the level it should be for the current content would create an imbalanced progression system and didn't feel like the right move for most players I've spoken with.

- **Loot and gearing.**: If you want to raid mythics for points for a specific piece of gear you keep losing rolls on (or maybe doesn't even drop), mythic points provided an alternate pathway to earn gear to improve your character. This would be a positive part of the experience to reward the more dedicated raid teams with mythic points, without neglecting casual players/pugs only clearing normal content.

- **Tuning.** The tuning on Phase 1 raids for Gruul/Mag was done really well. Casuals players, pugs, and dedicated raiders were able to clear the content and experience the raids. The mechanics being correct is what makes the fights feel the most authentic and engaging. Ideally for normal modes, CC staff would find the amount of health where mechanics have to be respected to complete the fight without it feeling totally cheesy. With that baseline, whatever is needed to tune for mythics can be done to make the encounters more challenging. 

- **Progression.** One thing our guild really enjoyed in Phase 1 was the sense of progression between normal and mythic raids. On nights when we had bad attendance, we could still clear the normal raids easily enough without a full roster. With full attendance, we could easily clear the mythic raids for points towards loot we needed for the raid.

- **Accessible content.** Having a normal raid mode doesn't hurt mythic players. Having a choice how players approach raid content gives everyone an opportunity to contribute to the mission of the server and improve the overall experience.

- **The system already exists.** Players generally know how the mythic point system works and the vendors are already in the game. I don't imagine it would be very difficult to create another menu of options for SSC and TK loot.

I hope this helps address some of the points Shael was making, and provides more context why having two raid difficulties could improve the quality of life for everyone looking to raid on CC.

Thanks for reading!
EliteCow

**Reply by balleny** [reply]
A system might be in place, but far from thoughtfully implemented imo: being able to "buy" loot from a boss that you never defeated in a mythic mode (unlocking an buy option) seems very unreasonable to me.

**Reply by EliteCow09** [reply]
I don't particularly care about the mythic points for loot, but agree if it's implemented there should be a check if the content has been cleared before you can purchase loot from it. For example, if there was a way to check if the achievement for SSC or TK is completed before that loot was available from the vendor. I don't know what the best solution is for this, just trying to offer a constructive solution.

I agree it would be an unfair advantage to buy Vashj loot without ever completing the boss. My suggestions are less about loot and more about game play and accessibility to content (for players of all abilities or time to dedicate to the server). Not everyone can spend multiple days progressing content because they have things going on outside the game. I'm just advocating for a way to bring the community into the process a bit more easily and let the content be accessible for those with limited time or energy towards playing on CC. You still should have to complete the content to get the gear though 100%.

**Reply by Jpizzle31** [reply]
This is a private test realm. Nothing about it is standard or normal. If you don't like it, then don't use the points. Mythic mode is there for those who want to be tryhards or possibly for a smaller guild that wants to be hardcore. You also have the option of normal Blizz-like TBC rates. Mythic gives you the option to play out all the mechanics and earn points to get gear, or you don't have to spend those points and just let them accumulate. It also provides an opportunity for those who don't have 2 1/2 - 3 hours on raid night to get their friends together, drink some beers, hang out, and play a 20-year-old game. All of this aligns with the ideal philosophy of Chromie Craft, taking a 3.3.5a client through mechanics and bug reports to establish a successful core for future servers. However, it seems like bug reporting is currently at an all-time low. The player base may be split or not even happy with the current process, leading them to give up and go to different games. This is unfortunate because CC has something great going for it.

**Reply by balleny** [reply]
the achievement would be available on regular kills already.
i am focusing on restricting the "tokens" earned to the specific bosses. if you didnt kill vashj x times in mythic, why should you be eligible to buy her loot.

**Reply by balleny** [reply]
chromie dropped distinctions between mythic and regular mode 3 months ago for new content. there is only 1 size fits all for anything released since kara. that is what is being discussed here.

bug reporting, triaging and fixing is on an very very low state right now. so much I agree.
Where there more people actively involved in the past? Sure!
Did people stop for private reasons or systematic reasons? Maybe, involved people stopped involving them, because the server responsibles "forced" their view on them. I can only speak so much to it that I the majority of people I talked with, who used to be active bug reporters, triagers or "devs", didnt quit for personal reasons.

**Reply by EliteCow09** [reply]
I don't disagree, but the bigger goal of this thread is less about the loot/point system and more about how creating two distinct experiences for raids could help the community experience the content.

Whatever the decision would be for implementing mythic points (if it does return) is an important one to get right. I wouldn't want to reward players unfairly for kills they have not earned or whatever the case may be as mentioned above. I would also be happy moving forward with two raid modes even if the mythic points are not implemented in any way. It's really about what the community wants.

The discussion here might be something that CC staff can further investigate and gather feedback on so we have a better understanding what the community wants from raids (and why). Maybe I am totally wrong and the majority of people don't even want or care about varying difficulties anymore - I don't know. I only can help provide feedback for the players around me and make a friendly suggestion.

One last thing I'll add here for CC staff would to be to be as specific as possible when collecting feedback from the community, so we're giving players the best chance to advocate for the experience they want.

**Reply by Nyeriah** [reply]
> Did people stop for private reasons or systematic reasons? Maybe, involved people stopped involving them, because the server responsibles "forced" their view on them. I can only speak so much to it that I the majority of people I talked with, who used to be active bug reporters, triagers or "devs", didnt quit for personal reasons.

@balleny Please open a ticket and elaborate on what circunstances are these.

**Reply by balleny** [reply]
without hijacking this discussion now too long, but I will not carry that discussion into non-public space.
Many of these things have been told in private and therefore i will not quote details, but an tldr would be: effort was put in, but feedback was _usually arbitrary_ overruled; the environment & decision finding; prioritizations.
(i am aware that this is obviously skewed simply given that this is based on the opinion of a distinct group - but it seemed reasonable given my experience with CC in the last 2 years

**Comment by Never6** [comment]
> Whatever the decision would be for implementing mythic points (if it does return) is an important one to get right. I wouldn't want to reward players unfairly for kills they have not earned or whatever the case may be as mentioned above. I would also be happy moving forward with two raid modes even if the mythic points are not implemented in any way. It's really about what the community wants.

I personally think that there  doesn't need to be 2 difficulty modes I just think tuning needs to be a lot better overall. Making bosses walking target dummies is quite frankly, bad tuning imo. This was a discussion a while back on if there should be mythic difficulty or not. I was one of the people that said there doesn't need to be.  Because T4 tuning was really good, bosses didn't die in 2mins but they werent meat sponges either. 
Morogrim is a great example on why i think the ssc tuning is bad and I am glad it was used.
The DPS brain in me absolutely loves the fight because I can do really well, its pure single target and he is quite literally a target dummy.
me actually playing the fight is just excuse my french, SHIT. I am doing literally nothing but DPS rotation for 8mins that isnt fun, that isnt engaging to anyone. 
reiterating this again, IMO there doesn't need to be 2 difficulties. Tuning just needs to be better

**Reply by balleny** [reply]
TBC just used to have 1-2 bosses each content tier which required damage dealers to pull their weight, to finish a boss before healers run short on mana. SSC tuning has been great as a healer, because you actually get to utilize your skills and dont finish a fight with 60% mana left.

Overall, this is the theme for all of classic, tbc and wotlk. Content isnt challenging, it is just time-consuming.
Imo, this only got better with very late ICC and partially cata.

**Reply by Never6** [reply]
well imo, i think 10min fights should be saved for end of raid encounters. If every fight is 8+ minutes gear upgrades feel less rewarding.
You can make content engaging and challenging without making fights long. But this also might be a matter of perspective because I have yet to heal SSC i only DPS

**Comment by QuantaCosta** [comment]
I'm sorry, maybe it's just me getting this wrong? But is OP on one hand advocating to keep raid difficulty as close to classic tbc as possible but on the other hand asking to tune down T6 raid content as it might take more than 3 hours to clear it?

**Comment by NinesCC** [comment]
I disagree with this proposal for many reasons:

- In the testing / reporting side of things, Normal modes in Gruul and Magtheridon already let you skip mechanics, for example the week that the cubes were not working in Magtheridon, we just did normal and did no cubes and healed through the damage, no problem! Since we aren't even interacting with the main mechanic how will we be able to find problems with the scripting if everything is faceroll?

- Vashj has already been killed by pugs, the fights aren't really hard, except for Vashj that requires coordination. I do agree that the HP values are quite high in many instances, but they can be killed none the less by a casual group of players that raid logs. Clear times are already on the 3 hour mark on my guild even with most people on vacation and they will continually go down weekly as people gear up. If playing 3 hours a week is too hardcore for some people, maybe this isn't the correct game for them as it's meant to be a time sink.

- The token system alters the game in a big way. DST being buyable by running 4 weeks of mythics degrades it's status to a badge gear type of loot. DST is a staple of TBC and guilds would run Gruul weekly it until Sunwell release, nowadays people don't care about running it anymore. The game has already changed (positively or negatively depending on PoV) and making even more loot purchasable will change the game even more by removing the carrot that drives a lot of people to play the game.

Have you ever thought that gear being so easy to acquire because of normal T4s being free loot, might be the reason is so hard to get any new recruits in guilds nowadays? These changes are devaluing guilds into something unnecessary, since you can just join a pug, zug-zug everything while watching a movie and get the same results as people in guilds. If everyone can get everything with no effort, who will join any guilds?

The game was made to be social and loot is meant to be rare, this proposed shift into everyone must be BiS and no one is left behind lands into the Fun-Server territory which isn't really what we were promised by the project. The consequences of these radical changes to the core of the game are unpredictable. How about we just let the game be and not change it to accommodate a minority that seems not be very invested in the game anyways.

TL;DR: 1 tuning for all, simple, just like the OG game. It was already discussed when T4 loot piñata fiasco happened, lets not run in circles.

Edit: Whatever the CC Staff's chooses for the difficulty is fine as long as it aligns to their vision of the project, but I do think that there should be some consistency on the difficulty and stick to it, because is the change of it being "too easy, then challenging, then back to easy" that is making people struggle because then they have to abruptly adjust like this whole situation.

**Comment by Cooties-cc** [comment]
These threads are fun to join in on. 

Overview from what I could read condensed into 4 talking points seems to be : 
1. Bug Reporting Ratios
2. Accessibility
3. Entertainment Value
4. Loot Acquisition

These are the 4 cores I see talked about here when discussing about new tier raids, or how to handle them. Going through the cores I am going to expand them, and also give as short as possible stance about them. 

1) Bug Reporting Rations - _Covers anything related to github tickets, or DM tickets about potential problems with the content._

This is something people simply cannot control or influence to a degree they are expecting. Free Will reigns supreme here, and the will of the individual trumps any incentives involved trying to entice someones hand. Already mentioned was the reports accuracy as to how knowledgeable the players are reporting. Flooding the ticket base with very inaccurate data (via more incentives for reports) will only lengthen the process to address and fix correct bug reports. I will say, perhaps personal, that TBC was the start of further obfuscation of bug reports. Perhaps due to the prog branch included, but to have multiple "correct" sources all stemming from different timelimes of the game (original, pre-nerf, post-nerf, classic, community request, etc), it has 100% made bug reporting a roulette wheel of fixes. 

2) Accessibility - _This covers how accessible the new content will be. Ranging from difficulty of content, [avg] length required to complete said content, and how the content could be tuned to increase or decrease player completion. Some of these are mutual, some are not._ 

In terms of SSC, and perhaps future raids, the accessibility rating is something that is subjective. Since there is no singular stance on how many people should be able to complete a task, there cannot really be a singular answer as to how it should be handled. I personally see no problem with content not being cleared by some, as this adds to the prestige of those who have cleared content. If everyone got a trophy, then it turns hard work and dedication into a participation sticker. Again, no real solid fixes to this from what I've read, only suggestions that won't cater to everyone, and the few I did want personally (like points towards a bonus roll) are limited to client restrictions. One last thing to add about tuning related accessibility was that if devs could, should watch raid ID's attempt/wipe/kill %'s and use that as part of a meter to gauge content tuning. If majority are getting kills, and as weeks pass the clear times are shortening, then it should be viewed as a success. 

3) Entertainment Value - _In short, the balance between being fun and being not fun._ 

Fun. Subjective. There's not much here to levy as a pro or con to content. You cannot declare that your fun should overrule anothers fun, and vice versa. There is no single path to fun. The best I could see in terms of fun, since it seems to partially tie into accessibility & loot, is to take more feedback from players that _have participated_ in the content. This means listening to raiders of any clear % and take into consideration what their viewpoints are. One commonly brought up is Morogrimm kill time. To me, people see this boss as "un-fun", and specific tuning can fix this, rather than having to change large aspects of how the raid overall works. 

4) Loot Acquisition - _Covers how people want to get their dank purps_

Basically if you kill something, you get something in return. Can also be subjectively tied into entertainment value, and objectively into accessibility via raid power. Hard to tackle if people want to mess with it to shift the value of fun or prog, but I think should remain closer to how fast content is being released. We are given an outline of when new things are to come, obtaining gear _should_ be paced to that expectation. So far loot wise, we might be getting drops perhaps too fast. If you want to compare in terms of "old content; experienced players", then you can check on classic versions (as close to our prog as we can get imo) at : https://classic.warcraftlogs.com/zone/statistics/1010#metric=fightwipes&timespan=1000&boss=628&sample=7
The rate is not high in terms of full clears. It started to get logged on Sept 18th, 2021 (released on the 15th), and majority (>=50%) of logs didn't get full clear till Dec 18th 2021. That's 91 days, or about 13 raid resets before **HALF** the raiders got a single vash kill. That's a lot slower loot than we're seeing now, and I don't think we need systems that give **_more_**.

In all, bug reports are always going to be up to the individual to do. Either they will or won't, and adding incentives to report isn't great as it can lower accuracy and increase processing speeds, making actual fixes slower. All content should be assessed to the actual rate in which it there is pass/fail, and act accordingly on a more individual basis, rather than sweeping changes to the entire system.  This means it won't cater to the masses, nor cater to the individual, but reflect aspects of content that may be fun/unfun, and fix them according to accurate mechanics 1st with completion rates 2nd. However fights shouldn't be all pushovers, and the bosses should fully utilize all their mechanics over a respectable time so raiders can process what is right and wrong, as to also not flood raid IDs with loot. Being a tool 1st, and reward 2nd, loot should still be flowing at a rate that doesn't make obsolete current content before new content becomes the focus. All of this is obviously not a 1 time fix, and will take more dynamic approaches to fix. 

Some fun ideas I was also going to add onto existing as alternatives that may or may not be possible : 
- Instead of mythic kill points, give people mythic _boss kill_ points. When you kill a boss, you only get points towards that bosses loot table. So when you go to the vendor, it's broken up into bosses instead of armor types of all raid. This way you are only being rewarded towards boss kills you have completed, and are unable to buy items from bosses you haven't. [Cons - faster rate of gear obtained since "more" points overall]
- Have mythic bosses drop or reward some unique tokens that can be added to an cosmetic ingame "store" you see similar to hardcore players. Items would be w/e devs see acceptable or possible, if this would even be possible. 
- Instead of trying to make the baseline version easy, with mythic being the alternative, why not make the easy aspect the alternative. This could be a detuned or even undertuned version of the raid that gave fewer items from boss kills, that would act like an "LFR" version of the raid. If people were short on time, short on players, pugs, learning, dad guild 😏 , etc, this version could be switched to for fewer rewards given. It also wouldn't give the SSC chev on vash kill, as to not interfere with raiders pushing firsts. Again, if possible. 
- Having more open polls or public feedback on how people are reacting to the new content somewhere. A survey perhaps collected for the first month of the raid/content addressing issue topics chosen by devs done via google forms or what not. Could give more insight into areas of the raid/content that might need more attention or players think need it, and direct a more positive fix for the server. 
- Perhaps do more community events related to new content? Have contests that are non-competitive (best SSC screenshot, vid, goofy w/e, etc) that add to player participation. Though rewards shouldn't be insane or if any outside of recognition, but could drive more ppl to passively participate in content that might lead to more fixes or increase player satisfaction.

**Comment by vvranceanu** [comment]
How much easier than pugs clearing the raids must it be?... The "social experiment" is a human nature experiment in which i can tell you the result right now, no need to run it: at the first sign of difficulty, people will say "fuck it" and do the easy version, either cause of bugs, mechanics they cant pass, performance, raid composition, raid attendance.

I am not in "full hardcore" guild. Even for ssc we sometimes wonder if 25 people will be able to attend. We struggled a bit on Vashj and that sparked lots od debates, discussions, going on ptr and generally a lot of fun. After we finally cleared it the sense of accomplishment and relief was so rewarding. The fight is now fun.

Please stop blaming the exterior factors and start looking in to your guild numbers, attendance, performance, etc. Please have the respect for others to wait 2 weeks after release and check if a pug cleared, even with 1/2 bosses remaining before claiming the content is too hard... the first fight in which everyone has some responsability and there is some raid mechanics that make raiding actually fun, i start to hear complaining that the tuning isn't right.

With all due respect, i hope this post isn't taken seriously

P.S.: Please understand you are not playing TBC, you are not playing WOTLK, you are not playing Blizzard WoW, this is a custom expericence, a new game, new rules if you want, new challenge. As long as it's not demonstrable impossible or close to, you should adapt to it and enjoy it for that it is. You know M'uru, for example, is notoriously hard, many didn't defeat it. Vashj, TK Kael, also first non tank and spank fights. Hijal will be walk in the park again. It's ok to have difficulties, just blame the 1.6k full epic dps first though, it's all i ask. Reading a basic guide on your class is not min-maxing
